data/json/items/comestibles/meat_dishes.json

```diff
@@ -69,6 +88,26 @@
     "flags": [ "EATEN_HOT" ],
     "fun": 8
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "bratwurst_sausage",
+    "name": "bratwurst",
+    "name_plural": "bratwursts",
+    "weight": 220,
```

Comment:
i suggest you change this to 100, to match the calorie value

Reply:
Done